packages/linux/package.mk

```diff
@@ -223,7 +217,7 @@ makeinstall_target() {
   if [ "$BOOTLOADER" = "u-boot" ]; then
     mkdir -p $INSTALL/usr/share/bootloader
     for dtb in arch/$TARGET_KERNEL_ARCH/boot/dts/*.dtb; do
-      cp $dtb $INSTALL/usr/share/bootloader 2>/dev/null || :
+      cp -v $dtb $INSTALL/usr/share/bootloader
```

Comment:
This change will break builds on platforms where dtb files are located in subfolders

```
cp: cannot stat 'arch/arm64/boot/dts/*.dtb': No such file or directory
Makefile:12: recipe for target 'image' failed
make: *** [image] Error 1
```

Suggest something like:
```
for dtb in arch/$TARGET_KERNEL_ARCH/boot/dts/*.dtb arch/$TARGET_KERNEL_ARCH/boot/dts/*/*.dtb; do
  if [ -f $dtb ]; then
    cp -v $dtb $INSTALL/usr/share/bootloader
  fi
done

Reply:
sure